test/topology_random_failures/suite.yaml

```diff
@@ -0,0 +1,11 @@
+type: Topology
+pool_size: 4
```

Comment:
because now test.py parallel tests run per file. pool size can be 1 now

Reply:
done